client/me/notification-settings/push-notification-settings/index.jsx

```diff
@@ -613,6 +616,7 @@ class PushNotificationSettings extends React.Component {
 		);
 
 		return (
+			/* eslint-disable wpcalypso/jsx-classname-namespace */
 			<Dialog
 				isVisible={ this.props.showDialog }
 				className=".notification-settings-push-notification-settings__instruction-dialog"
```

Comment:
why does the classname start with a `.`? :)

Reply:
Good spot, fixed :wink: